spec/griddler/email_spec.rb

```diff
@@ -432,6 +432,15 @@ def email_with_params(params)
     expect(headers[header_name]).to eq header_value
   end
 
+  it 'handles a hash being submitted' do
+    header = {
+              "X-Mailer" => "Airmail (271)",
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.